db/common/urbo_upsertmetadata.sql

```diff
@@ -0,0 +1,419 @@
+--
+-- Copyright 2017 Telefónica Digital España S.L.
+--
+-- This file is part of UrboCore API.
+--
+-- UrboCore API is free software: you can redistribute it and/or
+-- modify it under the terms of the GNU Affero General Public License as
+-- published by the Free Software Foundation, either version 3 of the
+-- License, or (at your option) any later version.
+--
+-- UrboCore API is distributed in the hope that it will be useful,
+-- but WITHOUT ANY WARRANTY; without even the implied warranty of
+-- MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+-- General Public License for more details.
+--
+-- You should have received a copy of the GNU Affero General Public License
+-- along with UrboCore API. If not, see http://www.gnu.org/licenses/.
+--
+-- For those usages not covered by this license please contact with
+-- iot_support at tid dot es
+--
+
+/*
+* General function to upsert metadata information at general level or for an specific scope.
+
+--------------------------------------------------------------------------------
+HOW TO USE:
+PgSQL:
+SELECT urbo_upsertmetadata(json);
+
+json Example:
+{
+  "schools.institute.position": {
+      "var_name" : "Posición del instituto"
+    },
+  "irrigation.humiditysensor": {
+      "table_name": "irrigation_humiditysensor",
+      "mandatory" : true
+    },
+  "dumps": {
+      "config": { "carto": { "account": "cedus-admin" } },
+      "nodata": false
+    }
+}
+Call Example:
+SELECT urbo_upsertmetadata('{"schools.institute.position":{"var_name":"Posición del instituto"},"irrigation.humiditysensor":{"table_name":"irrigation_humiditysensor","mandatory":true},"dumps":{"config":{"carto":{"account":"cedus-admin"}},"nodata":false}}'::json);
```

Comment:
A small mistake in the example: The function expects a `jsonb` type.
```suggestion
SELECT urbo_upsertmetadata('{"schools.institute.position":{"var_name":"Posición del instituto"},"irrigation.humiditysensor":{"table_name":"irrigation_humiditysensor","mandatory":true},"dumps":{"config":{"carto":{"account":"cedus-admin"}},"nodata":false}}'::jsonb);
```